### DISCUSSION TITLE
v3.0 inverse of properties needed to identify contracts from one single tender

### DISCUSSION THREAD

**Discussion by guascce** [eprocurementontology/eprocurementontology#523] [Ideas]
SubmissionStatisticalInformation epo:concernsSubmissionsForLot Lot  does not have the inverse property, so the only choice is to start from here.
Yet it is needed to search only within a given time frame, usually set by the Notice publication date. 
It should be possible to navigate from Lot to SubmissionStatisticalInformation, so that the limitations for querying (CAN,procedure, Lot)  are set before accessing statisticalInfo.

A second reverse property is needed between Lot and Contract.
Now Contract includesLot , to identify contracts with one tender only, it is needed to access Contract from Lot. Lot "resultsIn" for instance.

**Comment by costezki** [comment]
If there is a relationship between two entities in the knowledge base, we can query it in both directions, regardless of the direction used in the data. For example in the case of statistical information a query may look like this:

```
select *
{ 
  ?lot a epo:Lot .
  ?statisticalInformation epo:concernsSubmissionsForLot ?lot .
}
```

we do not need the inverse, such as epo:hasStatisticalInformation in order to keep the ?lot in the subject position. 

```
select *
{ 
  ?lot a epo:Lot .
  ?lot epo:hasStatisticalInformation  ?statisticalInformation .
}
```
However, if I misunderstood your questions, please provide a specific example and we will write a query to cover that.

**Comment by guascce** [comment]
OK. Thank you. Accepted as it is possible. Yet a question: can't it create capacity outages. May be it will be interesting to test some queries where the resolution uses the inverse link of the one that would be logical.

**Comment by giorgialodi** [comment]
Although you can also write queries not having an inverse property, inverse properties can be important in the development of an ontology. I am not a fan of introducing an inverse property for all the properties of the ontology, but when it is useful, why not?
